src/main/scala/ai/privado/languageEngine/javascript/processor/JavascriptProcessor.scala

```diff
@@ -125,10 +129,7 @@ class JavascriptProcessor(
       .withIgnoredFilesRegex(ruleCache.getExclusionRegex)
 
     val xtocpg = new JsSrc2Cpg().createCpg(cpgConfig).map { cpg =>
-      println(
-        s"${TimeMetric.getNewTime()} - Base processing done in \t\t\t\t- ${TimeMetric.setNewTimeToLastAndGetTimeDiff()}"
-      )
-
+      statsRecorder.endLastStage()
       applyDefaultOverlays(cpg)
```

Comment:
Need to start a stage before `applyDefaultOverlays(cpg)` and end after it.

We need to also check for this frontend to enable sub stage stats logging flag.